tests/Api/Admin/CustomersTest.php

```diff
@@ -64,6 +64,33 @@ public function it_gets_customer(): void
         );
     }
 
+    /** @test */
+    public function it_gets_customer_statistics(): void
+    {
+        $fixtures = $this->loadFixturesFromFiles([
+            'authentication/api_administrator.yaml',
+            'customer.yaml',
+            'channel.yaml',
+            'order/fulfilled_order.yaml',
+        ]);
+        $header = array_merge($this->logInAdminUser('api@example.com'), self::CONTENT_TYPE_HEADER);
```

Comment:
Would be cool to make use of recently added HeadersBuilder https://github.com/Sylius/Sylius/blob/1.13/tests/Api/Utils/HeadersBuilder.php